src/benchmarklib/abstract_benchmark_item_runner.hpp

```diff
@@ -41,6 +41,11 @@ class AbstractBenchmarkItemRunner {
   // Set the SQLite wrapper used for query verification. `nullptr` disables verification. Default is disabled.
   void set_sqlite_wrapper(const std::shared_ptr<SQLiteWrapper>& sqlite_wrapper);
 
+  // Returns a mapping from item ID to its relative weight in the execution of the benchmark. Relevant for example in
+  // the TPC-C benchmark, where not all transactions are executed equally often. Returns an empty vector if all items
+  // are weighted the same
```

Comment:
Would be easy to add, but we would either have to (a) implement it in each runner, even if no weights are used or (b) have it create the vector each time and return it by value instead of returning a reference to a static vector. Furthermore, I like the check in `benchmark_runner.cpp` where `.empty()` tells us if we are dealing with weights at all.

Reply:
Let's keep it simpel then.